src/index.js

```diff
@@ -104,6 +106,10 @@ module.exports = function (babel) {
           });
 
           if (specifierPath) {
+            if (useEmberModule) {
+              return;
+            }
```

Comment:
Short circuiting here doesn't seem obviously correct. When we update (for my other comment) to ensure that we add an import when the option is set, we'll need to do _something_ in this branch (other than short circuiting).

Some options that pop into my mind:

* Rename the local to `Ember` (may cause a conflict, if that is used for another local variable name) but don't remove the import specifier
* Use the local name for the other imports that are rewritten
* Use a dynamically generated local name (always) for all imports, renaming this specific specifier to use that dynamic local name and update all emitted replacements (for other imports) to use `_DynamicEmberGlobalThingyHere.Component` (instead of `Ember.Component`)

Generally speaking, I prefer the last option above...